pkg/apis/options/logging.go

```diff
@@ -12,7 +12,8 @@ type Logging struct {
 	RequestEnabled  bool           `flag:"request-logging" cfg:"request_logging"`
 	RequestFormat   string         `flag:"request-logging-format" cfg:"request_logging_format"`
 	StandardEnabled bool           `flag:"standard-logging" cfg:"standard_logging"`
-	StandardFormat  string         `flag:"standard-logging-format" cfg:"standard_logging_format"`
+	ErrToStdout     bool           `flag:"errors-to-stdout" cfg:"errors_to_stdout"`
+	StandardFormat  string         `flag:"standard-logging-format" cfg:"_format"`
```

Comment:
I don't think `cfg` was intended to change here? Should be left as `standard_loggin_format`?

Reply:
totally... ><
Will fix that later today.

Reply:
Nit, can we reverse the order here? All the enabled and format options were paired before, this breaks that pairing